builtins/en/facets/partials/instructions/fix-output-common.md

```
@@ -0,0 +1,8 @@
+## Work results
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)